src/NetAnalyzers/Core/Microsoft.NetCore.Analyzers/Runtime/PreferDictionaryContainsMethods.cs

```diff
@@ -86,19 +88,26 @@ void OnOperationAction(OperationAnalysisContext context)
                 //          - If the property's name is "Values", it must match the type substituted for TValue in the IDictionary`2 instance.
 
                 //  Once we have an applicable Contains method and an applicable property reference, we search for an applicable ContainsXXX
-                //  method on the IDictionary`2 receiver. 
```

Comment:
Arf sorry I forgot there was trailing conflicts on roslyn-analyzer... Let me fix that.